prov/gni/src/gnix_fabric.c

```diff
@@ -527,12 +528,14 @@ GNI_INI
 		gnix_xpmem_disabled = true;
 
 	/*
-	 * if for some reason we can't even allocate a single nic, bail.
+	 * well if we didn't get 1 nic, that means we must really be doing
+	 * FMA sharing.  Emit a warning message if FI_LOG_LEVEL appropriately
+	 * set.
```

Comment:
I don't think you need a comment stating that you're printing a warning message.